Editor/MenuInstallHook.cs

```diff
@@ -45,6 +46,27 @@ public void OnPreprocessAvatar(GameObject avatarRoot, BuildContext context)
                 avatar.expressionsMenu = menu;
                 context.ClonedMenus[menu] = menu;
             }
+            else
+            {
+                IEnumerable<VRCExpressionsMenu> GetMenus(VRCExpressionsMenu menu)
+                {
+                    if (!menu) yield break;
+                    foreach (var m in menu.controls
+                        .Where(control => control.type == VRCExpressionsMenu.Control.ControlType.SubMenu)
+                        .SelectMany(control => GetMenus(control.subMenu)))
+                        yield return m;
+                    yield return menu;
+                }
+                var currentMenus = GetMenus(avatar.expressionsMenu)
+                    .ToDictionary(menu => ObjectRegistry.GetReference(menu), menu => menu);
```

Comment:
If an expressions menu is cloned multiple times, and installed as multiple submenus in the expressions menu graph, this will throw an exception.

Reply:
I would suggest instead changing `VirtualMenu._targetMenuToInstaller` to use `ObjectReference` as its key; this would remove the need to change the menu installers, and should cleanly handle the case of multiple clones of the same menu (in that they will all install all items referenced by relevant Menu Installers).

Reply:
https://github.com/bdunderscore/modular-avatar/blob/c11a76642cb852897dd7eaf1612adae80d9267a9/Editor/Menu/VirtualMenu.cs#L309-L322

Reply:
Dictionaryの生成方法を変更しました。

> I would suggest instead changing VirtualMenu._targetMenuToInstaller to use ObjectReference as its key;

ObjectReferenceから得られるObjectはクローン前のObjectであり、それをキーにするとその後の処理がうまく動かなかったため_targetMenuToInstallerに全てのクローン後のObjectを登録する形で実装してみました。

Reply:
> ObjectReferenceから得られるObjectはクローン前のObjectであり、それをキーにするとその後の処理がうまく動かなかったため_targetMenuToInstallerに全てのクローン後のObjectを登録する形で実装してみました。

ObjectReferenceはクローン前後すべて同じObjectReferenceになるはずですが、キーにするとどのような不都合が出たのでしょうか？

Reply:
こちらで取得されるキーがObjectReferenceになりますが、ObjectReferenceからVRCExpressionsMenuを取得しようとするとそれはクローン前のものであるため正しくインストールされません。
https://github.com/bdunderscore/modular-avatar/blob/5090d45cfe7269e458dabdd959cab3cdb5005106/Editor/Menu/VirtualMenu.cs#L345-L352

……と考えましたが、レビューコメントの解釈が誤っていました。こちらは不要なコミットが多いので再度Pull Requestを送ります。